Tests/conf.json

```diff
@@ -1889,6 +1889,10 @@
             "playbookID": "Carbon Black Response Test",
             "fromversion": "5.0.0"
         },
+        {
+            "integrations": "carbonblack-v3",
+            "playbookID": "CarbonBlackEdr_Test"
+        },
```

Comment:
if it is mention in the yml of the integration, this section is not needed unless there are any special timeouts/pid_threshold

Reply:
validate said it is needed..